flux/stdlib/influxdata/influxdb/buckets.go

```diff
@@ -16,8 +16,23 @@ import (
 	"github.com/influxdata/influxql"
 )
 
+const BucketsKind = "influxdata/influxdb.localBuckets"
+
 func init() {
-	execute.RegisterSource(influxdb.BucketsKind, createBucketsSource)
+	execute.RegisterSource(BucketsKind, createBucketsSource)
+	plan.RegisterPhysicalRules(LocalBucketsRule{})
```

Comment:
Previously registering the `source` was good enough, but after the flux upgrade we also need to register a rule to rewrite a remote buckets call with no host to a local buckets call.